### PR TITLE
Add test cases for JSON resource marshaling

### PR DESCRIPTION
**Pull request by sagar23sj**
Resource Covered :

* TeamChange
* Page

Helps with issue: #55

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @sagar23sj !
LGTM.

Awaiting second LGTM before merging.

By the way, is there any reason why you don't just combine all the changes for a single file into one PR?

**Comment by sagar23sj** [comment]
I generally do it that way only @gmlewis , but there are a lot of JSON resources in event_types.go file and it was taking time good amount of time to even test few of them. So i decided to  test for 4-5 resources in a single PR, so that i don't get blocked for a long time since i find time to do this in between my regular work schedule.

**Comment by gmlewis** [comment]
OK, that's fine. It seems like GitHub was having merge issues before on the multiple additions to the single file. We'll see how it goes this time. 😁

**Pull request closed without merging**
